ports/supergaelicdodgeball/port.json

```
@@ -0,0 +1,30 @@
+{
+    "version": 1,
```

this should be version 3.
```suggestion
    "version": 3,
```